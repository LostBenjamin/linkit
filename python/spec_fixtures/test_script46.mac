

(((template 3 [10, 20, 30] [11, 22, 33])))

[[template A B C
    A,B,C
]]
