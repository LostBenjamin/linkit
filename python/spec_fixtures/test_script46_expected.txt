0,10,11:1,20,22:2,30,33

